bindings/c/tag_c.h

```diff
@@ -292,6 +292,27 @@ typedef enum {
 
 TAGLIB_C_EXPORT void taglib_id3v2_set_default_text_encoding(TagLib_ID3v2_Encoding encoding);
 
+/******************************************************************************
+ * Properties API
+ ******************************************************************************/
+
+/*!
+ * Sets the property with value.  value=null to remove, append to existing property or to replace
```

Comment:
Please try to have not more than 100 characters per line, I would even prefer 80. And use Doxygen `\return` and `\a` as in the other API comments.